src/EnvironmentInterface.php

```diff
@@ -75,4 +76,11 @@ public function getInlineRenderersForClass(string $inlineClass): iterable;
      * @return string
      */
     public function getInlineParserCharacterRegex(): string;
+
+    /**
+     * @param AbstractEvent $event
+     *
+     * @return AbstractEvent The processed event
+     */
+    public function dispatch(AbstractEvent $event): AbstractEvent;
```

Comment:
Why return the event vs void?

Reply:
I was thinking for ease of chaining:

```php
$foo = $environment->dispatch($event)->getFoo();
```

But I suppose that's unnecessary :)